alpine/license.md

```diff
@@ -0,0 +1 @@
+View [license information](https://github.com/gliderlabs/docker-alpine/blob/master/LICENSE) for the software contained in this image.
```

Comment:
Is this the license for the software contained in the image (ie, [Alpine Linux]()), or the license for the `Dockerfile` etc used to build the image? :wink: